examples/terraform/config.tf

```diff
@@ -0,0 +1,10 @@
+provider "aws" {
+  region = var.region
+
+  default_tags {
+    tags = {
+      Project     = var.project
+      Environment = var.environment
+    }
+  }
```

Comment:
This might be a style choice, but I like having resource-level tags to keep things more visible, but I 100% agree it can add some complexity. That being said, we use https://github.com/bridgecrewio/yor a LOT in our CI, and it does a pretty good job at making sure tag-able resources have tags, but there's a lot of room for improvement with how custom/default tags are done. 

It would be nice if we could mix and match, use default tags for project-level info like billing group/environment and then use yor for resource-level info like git blame.